### PR TITLE
impl FromIterator for Atom

### PR DESCRIPTION
**Pull request by burtonageo**

### CODE REVIEW

File: src/trivial_impls.rs [hunk 1]
```diff
@@ -12,6 +12,7 @@ use crate::{Atom, StaticAtomSet};
 use serde::{Deserialize, Deserializer, Serialize, Serializer};
 use std::borrow::Cow;
 use std::fmt;
+use std::iter::{FromIterator, IntoIterator};
```
Comment (nox) [inline src/trivial_impls.rs]: `IntoIterator` is already in prelude.

**Comment by nox** [comment]
Some of those impls seem a bit weird to me given we first need to collect to a `String` etc, I'll defer to @SimonSapin for taking a decision.

**Comment by SimonSapin** [comment]
What’s the motivation for adding these? They are all trivial compositions with `.collect::<String>()`, so it seems there is very little value in having them in this library.

**Comment by mrobinson** [comment]
No response here, so I'm just going to close this for now. Feel free to reopen if there is a good justification.

**Pull request closed without merging**